test/gamefield.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { GameField } from '../src/gamefield';
+
+const gameField = new GameField(' ', 10, 10);
```

Comment:
better to create a class instance before each tests instead of global variable

Reply:
Done